_config.yml

```diff
@@ -34,6 +34,7 @@ languages:
   bg: Български
   nl: Nederlands
   ro: Română
+  dk: Dansk
```

Comment:
I guess it should match with the dir name (https://github.com/semver/semver.org/tree/gh-pages/lang/da, dk vs. da), which is correct?

Reply:
@JohnTitor - yes, it should be 'da'